Configuration/Console/Commands.php

```diff
@@ -13,18 +14,19 @@
         \Helhum\Typo3Console\Command\CommandReferenceCommandController::class,
     ],
     'runLevels' => [
-        'typo3_console:install:databasedata' => \Helhum\Typo3Console\Core\Booting\RunLevel::LEVEL_MINIMAL ,
-        'typo3_console:install:defaultconfiguration' => \Helhum\Typo3Console\Core\Booting\RunLevel::LEVEL_MINIMAL ,
-        'typo3_console:install:*' => \Helhum\Typo3Console\Core\Booting\RunLevel::LEVEL_COMPILE ,
-        'typo3_console:cache:flush' => \Helhum\Typo3Console\Core\Booting\RunLevel::LEVEL_COMPILE ,
-        'typo3_console:commandreference:render' => \Helhum\Typo3Console\Core\Booting\RunLevel::LEVEL_MINIMAL ,
-        'typo3_console:configuration:*' => \Helhum\Typo3Console\Core\Booting\RunLevel::LEVEL_MINIMAL ,
+        'typo3_console:install:databasedata'         => \Helhum\Typo3Console\Core\Booting\RunLevel::LEVEL_MINIMAL,
+        'typo3_console:install:defaultconfiguration' => \Helhum\Typo3Console\Core\Booting\RunLevel::LEVEL_MINIMAL,
+        'typo3_console:install:fixfolderstructure'   => \Helhum\Typo3Console\Core\Booting\RunLevel::LEVEL_FULL,
```

Comment:
That is a breaking and also a rather severe change. It limits this command to be only executable with a full db connection, which isn't the case on a build system, which creates the package to be deployed. The command must stay very low level.